core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/messaginglayer/OutputManager.scala

```diff
@@ -170,13 +199,26 @@ class OutputManager(
     networkOutputBuffers.foreach(kv => kv._2.sendMarker(marker))
   }
 
-  def addPort(portId: PortIdentity, schema: Schema): Unit = {
+  def addPort(portId: PortIdentity, schema: Schema, storageUriOptional: Option[URI]): Unit = {
```

Comment:
first this is called `option` not `optional`. second, if scala you are using `Option[URI]`, can you align python for `Optional[URI]` or at least `Optional[str]`? currently you are using empty string on Python side. which is different from scala side.

Reply:
Changed variable name. Aligned Python implementation.